packages/react-components/react-tooltip/.npmignore

```diff
@@ -3,11 +3,10 @@
 bundle-size/
 config/
 coverage/
-docs/
+e2e/
```

Comment:
I also wouldn't expect to see `react-tooltip` changes.

Reply:
Yes! sorry the rebase fixed this. I tried to get this done on my laptop while I was OOF but something got lost in translation when I cloned the branch on that machine